vtr_flow/benchmarks/noc/Synthetic_Designs/Readme.txt

```diff
@@ -42,7 +44,11 @@ Running the benchmarks:
 		  -noc_routing_algorithm xy_routing -noc_swap_percentage 40 -number_of_seeds 5 -number_of_threads 1
 
 	        - The above command will generate an output file in the run directory that contains all the place and route metrics. This is a txt file with a name which matches the
-		  the flows file provided. So for the command shown above the outout file is 'complex_2_noc_1D_chain.txt'
+		    flows file provided. So for the command shown above the output file is 'complex_2_noc_1D_chain.txt'
+
+    Running VTR tasks:
+        - All synthetic benchmarks can be run as VTR tasks. Example tasks are provided in vtr_flow/tasks/noc_qor
+        - Instructions on how to run VTR tasks to measure QoR for NoC benchmarks in available in VTR Developer Guide.
```

Comment:
Might want to put a link to the appropriate part of the VTR developer guide here.